testsuite/integration-arquillian/test-apps/spring-boot-adapter-app/pom.xml

```diff
@@ -17,7 +17,7 @@
         <java.version>1.8</java.version>
         <jetty.version>${jetty94.version}</jetty.version>
 
-        <springboot-version>1.5</springboot-version>
+        <springboot-version>2.3</springboot-version>
```

Comment:
Is there any reason for this change? I am afraid this will result in wrong spring version in default matrix option in the pipeline. @miquelsi Do you have a Jenkins run with whole springboot matrix tested?